### DISCUSSION TITLE
run...docker-compose yaml files.....? Or at least store your last-used "Optional Settings" when running an Image

### DISCUSSION THREAD

**Discussion by wad11656** [docker/extension-ideas#188] [Ideas]
I'm honestly nearly speechless that i even am saying this. maybe I'm misunderstanding something, but I've been needing to manually re-input the same exact port and environment variables when I run my Image, every. single. time. Why not just...save the last data that was input? Or at least provide the option to do so? To build ontop of that idea, maybe create an extension that can run yaml docker-compose files, which allows passing in parameters like these

![image](https://github.com/docker/extension-ideas/assets/16929664/0eeb6984-ff90-4f41-a088-5a6dd0574e3f)

**Comment by benja-M-1** [comment]
Hi @wad11656 I am not sure to see how this is related to Docker extensions. IIUC you should probably create a feature request on https://github.com/docker/roadmap.

See https://github.com/docker/roadmap/blob/master/CONTRIBUTING.MD 🙂 
Have a good day!
